asv2/README.md

```diff
@@ -0,0 +1,111 @@
+
+* payments - send money directly to your friend's phone number!
+* social discovery - find someone's account based on their twitter!
+* any other identity applications!
+
+Here is a short demo of a payment from a user using [Kaala](https://kaala.app/) wallet to a user using [Libera](https://medium.com/impactmarket/ready-to-unlock-your-potential-meet-libera-your-new-crypto-wallet-d1053f917b95) wallet, using only a phone number:
+
+[<img width="800" alt="image" src="https://user-images.githubusercontent.com/46296830/207285114-6ef73be4-10f2-4afc-a066-811e1f3e1042.png">](https://www.loom.com/share/8afddd73ba324ec18aeb63fc96d568f9)
+
+## 🛠 How it Works
+
+ASv2 uses a federated model, meaning that anyone has the power to be an **issuer** of attestation mappings. Issuers have the freedom to decide how to verify that the user actually has ownership of their identifier. After verification, issuers register the mapping as an attestation to the [on-chain smart contract registry](https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/identity/FederatedAttestations.sol). Attestations are stored under the issuer that registered them. When looking up attestations, we then have to decide which issuers are trusted.
+
+Here are some active issuers verifying and registering attestations:
+
+| Issuer Name | Address|
+|-----|-----|
+|Kaala| 
+|Libera|
+|Valora
+
+Off-chain identifiers, originally in plaintext, are obfuscated before they are used in on-chain attestations to ensure user privacy and security. This is done with the help of the [Oblivious Decentralized Identifier Service (**ODIS**)](https://docs.celo.org/protocol/identity/odis). The details of the obfuscation process and how to interact with ODIS are described in the [docs about privacy](privacy.md).
+
+## 🧑‍💻 Quickstart
+
+The following steps use the Celo [ContractKit](https://docs.celo.org/developer/contractkit) to quickly set you up to play around with the protocol. If you would like to use a different library instead, please refer to the [example scripts](examples/).
+
+1. Add the [`@celo/identity`](https://www.npmjs.com/package/@celo/identity) package into your project.
+
+    ```console
+    yarn add @celo/identity
+    ```
+
+2. Derive the obfuscated identifier from your plaintext identifier. Refer to documentation on the [ODIS SDK](privacy.md#using-the-sdk) for detailed explanations on these parameters and steps.
+
+    ```typescript
+    import { OdisUtils } from "@celo/identity";
+    import { newKit } from "@celo/contractkit";
+
+    // create alfajores contractKit instance with your private key
+    const kit = await newKit("https://alfajores-forno.celo-testnet.org");
+    kit.addAccount(PRIVATE_KEY)
+
+    // approve and then send payment to OdisPayments to get quota for ODIS
+    const stableTokenContract = await kit.contracts.getStableToken();
```

Comment:
I'd be in favor of splitting out this "top up quota if necessary" step into it's own code snippet that also shows how to query remaining quota. I think this is more in line with how integrators will probably be building this (they wont be topping up quota before every request)